lib/gen_socket_client.ex

```diff
@@ -321,16 +319,24 @@ defmodule Phoenix.Channels.GenSocketClient do
   end
 
   defp reinit(state) do
-    :ets.delete_all_objects(state.message_refs)
+    :erlang.erase()
```

Comment:
Maybe we could get all the keys and delete where the first argument is the module name?